File: LeetCode/9/9_1.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        y = x
        z = 0
        if x < 0:
            return False
        while y > 0:
            z = z * 10 + y % 10
            y = y / 10
        if x == z:
            return True
        else:
            return False

File: LeetCode/9/9_1.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0) return false;
        int y = x, z = 0;
        while(y > 0)
        {
            z = z * 10 + y % 10;
            y /= 10;
        }
        return x == z;
    }
};
